spec/core_functions/color/blackness.hrx

```diff
@@ -36,6 +58,17 @@ a {
   b: 49.8039215686%;
 }
 
+<===> middle/zero_whiteness/warning
+DEPRECATION WARNING: Individual color-channel functions defined globally or in the color module are deprecated in favor of the new color.channel() function.
+
+Recommendation: color.channel(color.hwb(0, 0%, 50%), blackness)
```

Comment:
@mirisuzanne because of https://github.com/sass/sass/blob/colors-2/proposal/color-4-new-spaces.md#modified-color-module-functions I think this would eventually need to be updated to 

`Recommendation: color.channel(hwb(0, 0%, 50%), blackness)`

but when would that need to happen?

Reply:
Also, I don't think the commas are needed (anymore) so I can remove those.